features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,163 @@
+@clusterlogging
+@commonlogging
+Feature: Logging smoke test case
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-37508
+  @admin
+  @destructive
+  Scenario: One logging acceptance case for all cluster
+# Deploy cluster-logging operator via web console
+    Given logging service is removed successfully	
+    Given the logging operators are redeployed after scenario	
+    Given logging channel name is stored in the :logging_channel clipboard	
+    Given I register clean-up steps:	
+    """	
+    Given logging service is removed successfully	
+    Then the step should succeed	
+    """	
+    Given "cluster-logging" packagemanifest's catalog source name is stored in the :clo_opsrc clipboard	
+    Given "elasticsearch-operator" packagemanifest's catalog source name is stored in the :eo_opsrc clipboard	
+
+    Given I switch to the first user	
+    Given the first user is cluster-admin	
+    Given I open admin console in a browser	
+    # subscribe cluster-logging-operator	
+    When I perform the :goto_operator_subscription_page web action with:	
+      | package_name     | cluster-logging     |	
+      | catalog_name     | <%= cb.clo_opsrc %> |	
+      | target_namespace | openshift-logging   |	
+    Then the step should succeed	
+    And I perform the :set_custom_channel_and_subscribe web action with:	
+      | update_channel    | <%= cb.logging_channel %> |	
+      | install_mode      | OwnNamespace              |	
+      | approval_strategy | Automatic                 |	
```

Comment:
Seems we need to add an action to click the checkbox `Enable operator recommended cluster monitoring on this namespace` in web console, otherwise, the monitoring won't monitor the metrics in the openshift-logging namespace.